src/wamr/WAMRWasmModule.cpp

```diff
@@ -231,51 +253,18 @@ int WAMRWasmModule::executeWasmFunction(const std::string& funcName)
                      boundFunction);
         throw std::runtime_error("Did not find named wasm function");
     }
-
     // Note, for some reason WAMR sets the return value in the argv array you
     // pass it, therefore we should provide a single integer argv even though
     // it's not actually used
     std::vector<uint32_t> argv = { 0 };
-
-    // Invoke the function
-    bool success = aot_create_exec_env_and_call_function(
```

Comment:
This function was never part of the public API, and I think its not even available anymore.

We use the public API function `wasm_runtime_call_wasm`.